Tribler/Test/Community/Multichain/statistics/test_page_rank.py

```diff
@@ -7,18 +10,58 @@
 
 class TestIncrementalPageRank(TestCase):
     """
-    As it is a Monte Carlo method, integration tests can only be basic.
+    As it is a Monte Carlo method, integration tests are often probabilistic in nature.
+    That is, they might fail on rare occasions.
     """
-    def setUp(self):
-        nr_of_nodes = 1000
-        nr_of_edges = 100000
+    def create_graph(self, nr_of_nodes, nr_of_edges):
+        """
+        Helper method to create a random graph.
+
+        :param nr_of_nodes: amount of nodes in the random graph
+        :param nr_of_edges: amount of edges in the random graph
+        :return:
+        """
         self.graph = gnm_random_graph(nr_of_nodes, nr_of_edges)
         self.page_rank = IncrementalPageRank(self.graph)
-
-    def test_max_min(self):
         self.page_rank.initial_walk()
         self.page_rank.count()
-        ranks = self.page_rank.get_ranks()
-        self.assertTrue(all(0 <= rank <= 1 for rank in ranks.values()))
-        total_rounding_errors_bound = 0.05
-        self.assertAlmostEqual(sum(ranks.values()), 1, delta=total_rounding_errors_bound)
+        self.ranks = self.page_rank.get_ranks()
+
+    def test_single_node(self):
+        """
+        A single node should have a 1.0 ranking.
+
+        :return:
```

Comment:
I'd suggest removing the :return: from the tests, as they do not give any more information.